internal/primitives/runtime/digest.go

```diff
@@ -11,12 +17,82 @@ type DigestItemTypes interface {
 
 // Digest item that is able to encode/decode 'system' digest items and
 // provide opaque access to other items.
-// TODO: implement this as scale.VaryingDataType
-type DigestItem any
+type DigestItem struct {
```

Comment:
Why do we need to duplicate this type? Can't we use one of the two everywhere?

Reply:
Will be unified as part of a future refactoring.

Reply:
Already discussed by DM in slack